### PR TITLE
#2501 

### PR DESCRIPTION
**Pull request by tamaina**
and some refactoring on mfm/html

### CODE REVIEW

File: src/mfm/html.ts [hunk 1]
```diff
@@ -14,44 +14,44 @@ export default (tokens: Node[], mentionedRemoteUsers: INote['mentionedRemoteUser
 
 	const doc = window.document;
 
-	function dive(nodes: Node[]): any[] {
-		return nodes.map(n => handlers[n.name](n));
+	function appendChildren(children: Node[], targetElement: any): void {
+		for(const child of children.map(n => handlers[n.name](n))) targetElement.appendChild(child)
```
Comment (acid-chicken) [inline src/mfm/html.ts]: Please use `for (` instead of `for(`.

Comment (acid-chicken) [reply src/mfm/html.ts]: And missing a semicolon.

Comment (tamaina) [reply src/mfm/html.ts]: なおしました

**Review by acid-chicken — APPROVED** [review]

**Comment by syuilo** [comment]
🙏

**Pull request merged**